src/business-logic/services/test/superfly-timeline-builder.spec.ts

```diff
@@ -34,8 +33,8 @@ const LOOKAHEAD_PRIORITY: number = 0.1
 const BASELINE_PRIORITY: number = 0
 const LOW_PRIORITY: number = -1
 
-describe('superfly-timeline-builder', () => {
-  describe('buildTimeline', () => {
+describe(`${SuperflyTimelineBuilder.name}`, () => {
```

Comment:
As per Anders comment on #10, string interpolation can be removed.
Same goes for the describe below.